.devcontainer/devcontainer.json

```diff
@@ -10,8 +10,22 @@
 			"extensions": [
 				"ms-vscode.cpptools-extension-pack",
 				"llvm-vs-code-extensions.vscode-clangd",
-				"cschlosser.doxdocgen"
-			]
+				"cschlosser.doxdocgen",
+				"ryanluker.vscode-coverage-gutters"
```

Comment:
Interestingly enough the extension Coverage Gutter depends on this extension, as it uses it for its preview functionality.

Reply:
You are right. 

I thought that I installed that extension myself in the dev container, because I used it some time before. 
But it gets also installed, when only the Coverage Gutter extensions gets specified in the devcontainer.json file.